NEWS.md

```diff
@@ -1,9 +1,49 @@
+* [MODORDERS-804](https://issues.folio.org/browse/MODORDERS-804) - Remove validation for editing fund distribution when related invoice is approved or paid
+* [MODORDERS-803](https://issues.folio.org/browse/MODORDERS-803) - Implement Edit Fund Destribution blocking action (FLOW #3)
+* [MODORDERS-801](https://issues.folio.org/browse/MODORDERS-801) - Check and correct encumbrance ID links for invoices with Open or Reviewed status (FLOW #4)
+* [MODORDERS-800](https://issues.folio.org/browse/MODORDERS-800) - Implement moving encumbered value from budget 1 to budget 2 (FLOW #2)
+* [MODORDERS-789](https://issues.folio.org/browse/MODORDERS-789) - Creating large amounts of polines performance issue
+* [MODORDERS-788](https://issues.folio.org/browse/MODORDERS-788) - Implement fetching related invoices on PUT PO Line method to be able to implement all futher flows
+* [MODORDERS-784](https://issues.folio.org/browse/MODORDERS-784) - Logging improvement - Configuration
+* [MODORDERS-783](https://issues.folio.org/browse/MODORDERS-783) - Implement Orders Export History API
+* [MODORDERS-780](https://issues.folio.org/browse/MODORDERS-780) - Replace FolioVertxCompletableFuture usage
+* [MODORDERS-775](https://issues.folio.org/browse/MODORDERS-775) - Add deduplication mechanism for Create Order event
+* [MODORDERS-773](https://issues.folio.org/browse/MODORDERS-773) - Implement action profile handler for order creation
+* [MODORDERS-724](https://issues.folio.org/browse/MODORDERS-724) - Replace use of deprecated class HttpClientFactory
+* [MODORDERS-696](https://issues.folio.org/browse/MODORDERS-696) - Replace all utility methods for HTTP calls from HelperUtils
+* [MODORDERS-647](https://issues.folio.org/browse/MODORDERS-647) - Implement Find or create logic for Holdings generated through the GOBI API
+* [MODORDERS-598](https://issues.folio.org/browse/MODORDERS-598) - Replace CompletableFuture.allOf with FolioVertxCompletableFuture.allOf
+
+### Bug Fixes
+* [MODORDERS-841](https://issues.folio.org/browse/MODORDERS-841) - Encumbrance current amount is calculated incorrectly after changing fund distribution when related paid invoice exists
+* [MODORDERS-833](https://issues.folio.org/browse/MODORDERS-833) - Negative encumbrance values appear for expense classes after rollover
+* [MODORDERS-827](https://issues.folio.org/browse/MODORDERS-827) - Order view is not restricted by Acquisition unit
+* [MODORDERS-816](https://issues.folio.org/browse/MODORDERS-816) - A user with order create permission can not add suffix and prefix to the order
+* [MODORDERS-810](https://issues.folio.org/browse/MODORDERS-810) - Invoice is unable to approve due to protected fields on productIds
+* [MODORDERS-713](https://issues.folio.org/browse/MODORDERS-713) - Add prefix and suffix to PO number when create or update order
+* [MODORDERS-473](https://issues.folio.org/browse/MODORDERS-473) - Memory leak in orders app
```

Comment:
no changes in master branch regarding MODORDERS-473. there were only perfomance checks

Reply:
I deleted this task from news